community.md

```diff
@@ -144,6 +144,18 @@ If you send us an email with a code snippet, make sure that:
 3. you keep formatting when pasting code in order to keep the code readable
 4. there are enough import statements to avoid ambiguities
 
+## Slack
+
+You can join the [Apache Flink community on Slack.](https://join.slack.com/t/apache-flink/shared_invite/zt-19uavkkkw-qwMimi9gawIv9Fz9xFuqrw)
+After creating an account in Slack, don't forget to introduce yourself in #introductions.
+In case the invite link no longer works, please reach out to the [Dev mailing list](#mailing-Lists). 
```

Comment:
```suggestion
Due to Slack limitations the invite link expires after 100 invites. If it is expired, please reach out to the [Dev mailing list](#mailing-Lists). 
```
Maybe it makes sense to make this more explicit, so that people understand why this happens?